src/WebJobs.Script/ScriptConstants.cs

```diff
@@ -184,6 +184,7 @@ public static class ScriptConstants
         public static readonly ImmutableArray<string> AssemblyFileTypes = ImmutableArray.Create(".dll", ".exe");
         public static readonly string HostUserAgent = $"azure-functions-host/{ScriptHost.Version}";
         public static readonly NuGetVersion ExtensionBundleVersionTwo = new NuGetVersion("2.0.0");
+        public static readonly long? DefaultMaxRequestBodySize = 104857600;
```

Comment:
Can you clarify again what API required you to define this as nullable?

Reply:
It is not required. I have made it long.